Wiki/Modules/FvwmPager/index.md

```diff
@@ -18,3 +18,71 @@ active windows are. Windows in the pager are shown in the same color as
 their fvwm decorations.  The pager can be used to change your viewport into
 the current desktop, to change desktops, or to move windows around.
 
+## Configuration and Use
+
+FvwmPager is used to show the layout of your desktop. It will show all the Pages
+and Desktops you have set up, and the windows opened in each.
+
+A single fvwm Desktop is split up into a grid of pages. The number of pages is
+configured with the DesktopSize command. For the FvwmPager configuration, refer
+to the man page for a full explanation of all the options available. This
+configuration is the same as in fvwm default config.
+
+![image](pager2x2.png){: style="float: right; margin-right: 5em; margin-top: 1em;"}
+
+{% highlight fvwm %}
+{% capture txt %}
+DesktopSize 2x2
+
+DestroyModuleConfig FvwmPager: *
+*FvwmPager: Geometry 150x150-0+0
+*FvwmPager: Font None
+*FvwmPager: Colorset * 10
+*FvwmPager: HilightColorset * 13
+*FvwmPager: BalloonColorset * 10
+*FvwmPager: WindowColorsets 10 11
+*FvwmPager: Balloons All
+*FvwmPager: BalloonFont "xft:Sans:Bold:size=8:antialias=True"
+*FvwmPager: BallonYOffset +2
+*FvwmPager: Window3dBorders
+*FvwmPager: MiniIcons"
+*FvwmPager: UseSkipList
+{% endcapture %}
+{{ txt }}
+{% endhighlight %}
```

Comment:
Do not use `{% capture %}` this way, there is no reason to have to capture this into a variable, in essence just remove the two `capture` lines and the `{{ txt }}`, the text inside these blocks is considered code, no formatting is done to it.

A note for the future (this isn't quite ready yet), but `{% highlight fvwm %}` is going to be replaced with `{% fvwm2rc %}` tags, as we are almost ready to get a new workflow that will add syntax highlighting back to the wiki.

Reply:
We just merged the workflow, we have syntax highlighting back, but you have to use `{% fvwm2rc %}...{% endfvwm2rc %}` to format blocks using the custom fvwm code lexer.

Reply:
> Do not use {% capture %} this way

In my search to add an image inside `{% highlight fvwm %}`, only that worked.
https://rasatpc.net/screenshot/wiki/image-highlight.png

If there is a better way, let me know.

Reply:
That is due to your floating image, and in general I would avoid using floats. For the most part images should all have the classes, `{:.mx-auto .d-block .img-fluid}`, though floats do have their use.

I have seen other cases where floats get in the way of the code blocks, I'll look into that, i probably need a clearfix at the end of the block (part of what makes floats annoying is needing clearfixes sometimes). Since we don't really use floating divs, it shouldn't be a problem.